bin/flutter.bat

```diff
@@ -80,6 +80,12 @@ GOTO :after_subroutine
   :do_sdk_update_and_snapshot
     ECHO Checking Dart SDK version...
     CALL PowerShell.exe -ExecutionPolicy Bypass -Command "& '%FLUTTER_ROOT%/bin/internal/update_dart_sdk.ps1'"
+    SET exit_code=%ERRORLEVEL%
+    IF %exit_code% NEQ 0 (
+      ECHO Error: Unable to update Dart SDK.
+      REM Do not use /B here, we want to exit out of the script, not just the subroutine
```

Comment:
This is the only place where it matters: we are inside a subroutine and we want to exit the entire script. In the other places where I removed the /B, it didn't actually change any logic because we weren't in a subroutine (I just removed it to be consistent: in case of error, exit the script).